taichi/backends/metal/kernel_utils.cpp

```diff
@@ -120,12 +120,16 @@ KernelContextAttributes::KernelContextAttributes(const Kernel &kernel)
     // Put scalar args in the memory first
     for (int i : scalar_indices) {
       auto &attribs = (*vec)[i];
+      const size_t dt_bytes = metal_data_type_bytes(attribs.dt);
+      bytes = (bytes + dt_bytes - 1) / dt_bytes * dt_bytes;
```

Comment:
Could you add a comment here explaining how this magic transformation works?

Reply:
I'm just thinking about what comment is most proper here. What about `Align bytes to the nearest multiple of dt_bytes`?

Reply:
This makes sense to me :)

Reply:
Done